go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/youngkin/go-leaderelection
```

Comment:
Also this probably needs to be replaced with `github.com/Comcast/go-leaderelection`

Reply:
Good catch, definitely!

Reply:
Done.